### PR TITLE
Removes limitation on root folder for editing in VSCode

### PR DESCRIPTION
**Pull request by ashishchoudhary001**
Removes limitation on root folder for editing in VSCode

### CODE REVIEW

File: src/client/PortalWebView.ts [hunk 1]
```diff
@@ -130,53 +126,65 @@ export class PortalWebView {
...
-            vscode.Uri.joinPath(uri as vscode.Uri, "web-files", "theme.css")
-        );
-        const theme = `<link href="${url}" rel="stylesheet" />`;
-
-        return html + bootstrap + theme;
+        const uri = PortalWebView.getPortalRootFolder();
+        if (uri) {
+            // Add bootstrap.min.css
+            let url = webview.asWebviewUri(
+                vscode.Uri.joinPath(
```
Comment (davidjenni) [inline src/client/PortalWebView.ts:133]: consistency nit: most other .joinPath() are single line, which I think is more readable (assuming the # of args is low)

File: src/common/PortalConfigFinder.ts [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        }
+    }
+    return null;
+}
+
+/**
+ * returns the workspaceRootFolder for the file being edited
+ * in case of multi-root workspaces this will return the real root folder
+*/
+// function getRootFolder(workspaceRootFolders: WorkspaceFolder[] | null, pathOfFileBeingEdited: string) : string | null{
```
Comment (davidjenni) [inline src/common/PortalConfigFinder.ts:72]: still needed? -> delete

**Review by davidjenni — APPROVED** [review]

File: src/client/PortalWebView.ts (hunk 1 above)
Comment (ashishchoudhary001) [reply src/client/PortalWebView.ts]: fixed

File: src/common/PortalConfigFinder.ts (hunk 2 above)
Comment (ashishchoudhary001) [reply src/common/PortalConfigFinder.ts]: yup not needed. Removed.

**Pull request merged**